snewpdag/plugins/gen/Combine.py

```diff
@@ -43,6 +43,8 @@ def alert(self, data):
     # histograms - only works if all of them have the same spec
     nodef = True
     for d in data['gen']:
+      if 'gen_t_delay' in d:
```

Comment:
I guess this time is the true time for the beginning of the neutrino signal (for the detector whose payload this is), rather than a time delay?

Reply:
yes, indeed, actually I usually call it t_true all along, I will change it in the generator where this name is given